app/helpers/admin_helper.rb

```diff
@@ -60,7 +60,32 @@ def indented_option_label(node, label_method)
   end
 
   # Displays Font Awesome icons
-  def icon_tag(class_name)
-    content_tag(:i, "", class: "fa fa-#{class_name}")
+  def icon_tag(class_name, options: {})
+    content_tag(:i, "", id: options[:id], data: options[:data], class: "fa fa-#{class_name} #{options[:extra_classes]}")
+  end
+
+  def documentation_popover(documentations, html_identifier: "")
+    documentation = documentations[html_identifier]
+    if documentation.present?
+      data_content = documentation&.summary_content.to_s
```

Comment:
I've not seen `&` used in this way before inside dot notation. This is intended and not a typo?

Reply:
it's the new safe navigator operator
`documentation&.summary_content` here means `documentation && documentation.summary_content`

it's pretty neat :)

Reply:
Yep, it's an alternative to doing `documentation.try(:summary_content).try(:to_s)`

Reply:
Love it. Thank you!